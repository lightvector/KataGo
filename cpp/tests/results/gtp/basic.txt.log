: GTP Engine starting...
: KataGo v1.8.2
: Using TrompTaylor rules initially, unless GTP/GUI overrides this
: Using 1 CPU thread(s) for search
: nnRandSeed0 = forTesting
: After dedups: nnModelFile0 = tests/models/g170-b6c96-s175395328-d26788732.bin.gz useFP16 false useNHWC false
: Cuda backend thread 0: Found GPU Tesla V100-SXM2-16GB memory 16945512448 compute capability major 7 minor 0
: Cuda backend thread 0: Model version 8 useFP16 = false useNHWC = false
: Cuda backend thread 0: Model name: g170-b6c96-s175395328-d26788732
: Loaded neural net with nnXLen 19 nnYLen 19
: Loaded config configs/gtp_example.cfg and/or command-line and query overrides
: Loaded model tests/models/g170-b6c96-s175395328-d26788732.bin.gz
: Config override: cudaUseFP16 = false
: Config override: cudaUseNHWC = false
: Config override: forDeterministicTesting = true
: Config override: logDir = 
: Config override: logFile = tests/results/gtp/basic.txt.log
: Config override: logTimeStamp = false
: Config override: maxPlayouts = 10000
: Config override: maxVisits = 100
: Config override: nnRandSeed = forTesting
: Config override: nnRandomize = false
: Config override: numSearchThreads = 1
: Config override: openclUseFP16 = false
: Config override: searchRandSeed = forTesting
: Model name: g170-b6c96-s175395328-d26788732
: GTP ready, beginning main protocol loop
: Controller: genmove_debug b
: MoveNum: 0 HASH: CDCBC1F514D7E680FACD226074256633
   A B C D E F G H J K L M N O P Q R S T
19 . . . . . . . . . . . . . . . . . . .
18 . . . . . . . . . . . . . . . . . . .
17 . . . . . . . . . . . . . . . . . . .
16 . . . . . . . . . . . . . . . . . . .
15 . . . . . . . . . . . . . . . . . . .
14 . . . . . . . . . . . . . . . . . . .
13 . . . . . . . . . . . . . . . . . . .
12 . . . . . . . . . . . . . . . . . . .
11 . . . . . . . . . . . . . . . . . . .
10 . . . . . . . . . . . . . . . . . . .
 9 . . . . . . . . . . . . . . . . . . .
 8 . . . . . . . . . . . . . . . . . . .
 7 . . . . . . . . . . . . . . . . . . .
 6 . . . . . . . . . . . . . . . . . . .
 5 . . . . . . . . . . . . . . . . . . .
 4 . . . . . . . . . . . . . . . . @ . .
 3 . . . . . . . . . . . . . . . . . . .
 2 . . . . . . . . . . . . . . . . . . .
 1 . . . . . . . . . . . . . . . . . . .

koPOSITIONALscoreAREAtaxNONEsui1komi7.5
Time taken: ###
Root visits: 100
New playouts: 100
NN rows: 95
NN batches: 95
NN avg batch size: 1
PV: Q17 D3 C16 Q3
Tree:
: T   0.07c W  -0.94c S   1.01c ( -0.1 L  +0.0) N     100  --  Q17 D3 C16 Q3
---Black(^)---
Q17 : T   1.44c W  -0.03c S   1.47c ( +0.2 L  +0.3) LCB   -5.74c P  8.73% WF  11.1 PSV      19 N      11  --  Q17 D3 C16 Q3
C4  : T   0.86c W  -0.60c S   1.47c ( -0.1 L  -0.0) LCB  -12.55c P  8.60% WF  10.1 PSV      10 N      10  --  C4 R16 C16
D3  : T   1.35c W  -0.06c S   1.41c ( +0.2 L  +0.3) LCB  -10.28c P  8.43% WF  10.1 PSV      10 N      10  --  D3 R16 C16
C16 : T   0.75c W  -0.59c S   1.34c ( -0.0 L  +0.1) LCB  -10.69c P  8.21% WF  10.0 PSV      10 N      10  --  C16 D3 Q3
Q3  : T   0.07c W  -0.70c S   0.77c ( +0.1 L  +0.2) LCB  -11.02c P  8.99% WF   9.0 PSV       9 N       9  --  Q3 R16 D3 D17
R4  : T  -0.89c W  -1.57c S   0.68c ( -0.2 L  -0.0) LCB  -12.30c P  8.90% WF   8.9 PSV       9 N       9  --  R4 R16 D3 D17
D17 : T  -0.30c W  -1.41c S   1.12c ( -0.1 L  -0.0) LCB  -12.72c P  8.00% WF   9.0 PSV       8 N       9  --  D17 D3 Q3 Q17
R16 : T   0.52c W  -0.61c S   1.14c ( +0.0 L  +0.1) LCB  -14.49c P  7.34% WF   8.0 PSV       8 N       8  --  R16 D3 C16
Q4  : T  -0.81c W  -1.40c S   0.58c ( -0.2 L  -0.1) LCB  -28.33c P  5.83% WF   5.0 PSV       5 N       5  --  Q4 D3 D17
D4  : T  -0.63c W  -1.33c S   0.70c ( -0.2 L  -0.0) LCB -106.30c P  4.67% WF   4.0 PSV       4 N       4  --  D4 R4 C16

: = R4
: Controller: genmove_debug w
: MoveNum: 1 HASH: F19857B2A2B00E3CA2C67ECEB0719FCA
   A B C D E F G H J K L M N O P Q R S T
19 . . . . . . . . . . . . . . . . . . .
18 . . . . . . . . . . . . . . . . . . .
17 . . . . . . . . . . . . . . . @ . . .
16 . . . . . . . . . . . . . . . . . . .
15 . . . . . . . . . . . . . . . . . . .
14 . . . . . . . . . . . . . . . . . . .
13 . . . . . . . . . . . . . . . . . . .
12 . . . . . . . . . . . . . . . . . . .
11 . . . . . . . . . . . . . . . . . . .
10 . . . . . . . . . . . . . . . . . . .
 9 . . . . . . . . . . . . . . . . . . .
 8 . . . . . . . . . . . . . . . . . . .
 7 . . . . . . . . . . . . . . . . . . .
 6 . . . . . . . . . . . . . . . . . . .
 5 . . . . . . . . . . . . . . . . . . .
 4 . . . . . . . . . . . . . . . . X1. .
 3 . . . . . . . . . . . . . . . . . . .
 2 . . . . . . . . . . . . . . . . . . .
 1 . . . . . . . . . . . . . . . . . . .

koPOSITIONALscoreAREAtaxNONEsui1komi7.5
Time taken: ###
Root visits: 100
New playouts: 91
NN rows: 80
NN batches: 80
NN avg batch size: 1
PV: R16 D3 D17 D5
Tree:
: T   0.36c W   0.41c S  -0.05c ( -0.1 L  -0.1) N     100  --  R16 D3 D17 D5
---White(^)---
R16 : T   1.11c W   1.20c S  -0.08c ( +0.0 L  -0.0) LCB   -1.84c P 13.96% WF  17.1 PSV      24 N      17  --  R16 D3 D17 D5
D3  : T   0.81c W   0.79c S   0.02c ( -0.1 L  -0.1) LCB   -2.41c P 13.89% WF  16.1 PSV      16 N      16  --  D3 Q17 D17 Q15
Q17 : T   0.54c W   0.52c S   0.01c ( -0.0 L  -0.1) LCB   -2.79c P 10.65% WF  12.0 PSV      12 N      12  --  Q17 D3 D17 D5
D17 : T   0.12c W  -0.03c S   0.15c ( -0.1 L  -0.1) LCB   -5.56c P 10.29% WF  11.0 PSV      11 N      11  --  D17 D3 Q17 D5
C16 : T   1.19c W   0.76c S   0.43c ( +0.1 L  +0.1) LCB   -4.66c P  9.04% WF  11.1 PSV      11 N      11  --  C16 C4 Q17 E4
C4  : T   0.24c W   0.12c S   0.13c ( -0.1 L  -0.1) LCB   -5.33c P  9.35% WF  11.0 PSV      10 N      11  --  C4 Q17 D17 Q15
Q4  : T  -0.85c W   0.24c S  -1.08c ( -0.1 L  -0.3) LCB  -30.28c P  6.62% WF   5.9 PSV       6 N       6  --  Q4 Q3 C4 C16
D4  : T  -0.59c W  -0.40c S  -0.19c ( -0.3 L  -0.4) LCB  -28.22c P  5.74% WF   5.0 PSV       5 N       5  --  D4 C16 Q17
Q16 : T  -0.82c W  -0.25c S  -0.56c ( -0.2 L  -0.3) LCB  -22.08c P  5.72% WF   5.0 PSV       5 N       5  --  Q16 D3 D17
D16 : T  -1.17c W  -0.73c S  -0.44c ( -0.3 L  -0.4) LCB  -26.89c P  5.61% WF   4.9 PSV       5 N       5  --  D16 D3 Q17

: = Q17
: Controller: clear_board
: = 
: Controller: genmove_debug b
: MoveNum: 0 HASH: CDCBC1F514D7E680FACD226074256633
   A B C D E F G H J K L M N O P Q R S T
19 . . . . . . . . . . . . . . . . . . .
18 . . . . . . . . . . . . . . . . . . .
17 . . . . . . . . . . . . . . . . . . .
16 . . @ . . . . . . . . . . . . . . . .
15 . . . . . . . . . . . . . . . . . . .
14 . . . . . . . . . . . . . . . . . . .
13 . . . . . . . . . . . . . . . . . . .
12 . . . . . . . . . . . . . . . . . . .
11 . . . . . . . . . . . . . . . . . . .
10 . . . . . . . . . . . . . . . . . . .
 9 . . . . . . . . . . . . . . . . . . .
 8 . . . . . . . . . . . . . . . . . . .
 7 . . . . . . . . . . . . . . . . . . .
 6 . . . . . . . . . . . . . . . . . . .
 5 . . . . . . . . . . . . . . . . . . .
 4 . . . . . . . . . . . . . . . . . . .
 3 . . . . . . . . . . . . . . . . . . .
 2 . . . . . . . . . . . . . . . . . . .
 1 . . . . . . . . . . . . . . . . . . .

koPOSITIONALscoreAREAtaxNONEsui1komi7.5
Time taken: ###
Root visits: 100
New playouts: 100
NN rows: 0
NN batches: 0
NN avg batch size: -nan
PV: Q17 D3 C16 Q3
Tree:
: T   0.07c W  -0.94c S   1.01c ( -0.1 L  +0.0) N     100  --  Q17 D3 C16 Q3
---Black(^)---
Q17 : T   1.44c W  -0.03c S   1.47c ( +0.2 L  +0.3) LCB   -5.74c P  8.73% WF  11.1 PSV      19 N      11  --  Q17 D3 C16 Q3
C4  : T   0.86c W  -0.60c S   1.47c ( -0.1 L  -0.0) LCB  -12.55c P  8.60% WF  10.1 PSV      10 N      10  --  C4 R16 C16
D3  : T   1.35c W  -0.06c S   1.41c ( +0.2 L  +0.3) LCB  -10.28c P  8.43% WF  10.1 PSV      10 N      10  --  D3 R16 C16
C16 : T   0.75c W  -0.59c S   1.34c ( -0.0 L  +0.1) LCB  -10.69c P  8.21% WF  10.0 PSV      10 N      10  --  C16 D3 Q3
Q3  : T   0.07c W  -0.70c S   0.77c ( +0.1 L  +0.2) LCB  -11.02c P  8.99% WF   9.0 PSV       9 N       9  --  Q3 R16 D3 D17
R4  : T  -0.89c W  -1.57c S   0.68c ( -0.2 L  -0.0) LCB  -12.30c P  8.90% WF   8.9 PSV       9 N       9  --  R4 R16 D3 D17
D17 : T  -0.30c W  -1.41c S   1.12c ( -0.1 L  -0.0) LCB  -12.72c P  8.00% WF   9.0 PSV       8 N       9  --  D17 D3 Q3 Q17
R16 : T   0.52c W  -0.61c S   1.14c ( +0.0 L  +0.1) LCB  -14.49c P  7.34% WF   8.0 PSV       8 N       8  --  R16 D3 C16
Q4  : T  -0.81c W  -1.40c S   0.58c ( -0.2 L  -0.1) LCB  -28.33c P  5.83% WF   5.0 PSV       5 N       5  --  Q4 D3 D17
D4  : T  -0.63c W  -1.33c S   0.70c ( -0.2 L  -0.0) LCB -106.30c P  4.67% WF   4.0 PSV       4 N       4  --  D4 R4 C16

: = C16
: Controller: genmove_debug w
: MoveNum: 1 HASH: 5CAAD553F3A4AF0F47459C51E40D103A
   A B C D E F G H J K L M N O P Q R S T
19 . . . . . . . . . . . . . . . . . . .
18 . . . . . . . . . . . . . . . . . . .
17 . . . . . . . . . . . . . . . . . . .
16 . . X1. . . . . . . . . . . . . @ . .
15 . . . . . . . . . . . . . . . . . . .
14 . . . . . . . . . . . . . . . . . . .
13 . . . . . . . . . . . . . . . . . . .
12 . . . . . . . . . . . . . . . . . . .
11 . . . . . . . . . . . . . . . . . . .
10 . . . . . . . . . . . . . . . . . . .
 9 . . . . . . . . . . . . . . . . . . .
 8 . . . . . . . . . . . . . . . . . . .
 7 . . . . . . . . . . . . . . . . . . .
 6 . . . . . . . . . . . . . . . . . . .
 5 . . . . . . . . . . . . . . . . . . .
 4 . . . . . . . . . . . . . . . . . . .
 3 . . . . . . . . . . . . . . . . . . .
 2 . . . . . . . . . . . . . . . . . . .
 1 . . . . . . . . . . . . . . . . . . .

koPOSITIONALscoreAREAtaxNONEsui1komi7.5
Time taken: ###
Root visits: 100
New playouts: 90
NN rows: 60
NN batches: 60
NN avg batch size: 1
PV: D3 Q3 Q17 Q5
Tree:
: T  -0.81c W  -0.61c S  -0.20c ( -0.4 L  -0.3) N     100  --  D3 Q3 Q17 Q5
---White(^)---
D3  : T  -0.21c W  -0.21c S   0.00c ( -0.4 L  -0.3) LCB   -3.11c P 13.58% WF  17.1 PSV      23 N      17  --  D3 Q3 Q17 Q5
R16 : T  -0.24c W  -0.12c S  -0.12c ( -0.3 L  -0.3) LCB   -3.42c P 13.06% WF  16.1 PSV      16 N      16  --  R16 D3 Q3 E16
Q3  : T  -0.22c W  -0.26c S   0.04c ( -0.4 L  -0.4) LCB   -3.80c P 11.98% WF  14.1 PSV      14 N      14  --  Q3 C4 Q17 E4
R4  : T  -0.87c W  -0.87c S   0.00c ( -0.5 L  -0.4) LCB   -5.44c P 11.35% WF  12.0 PSV      12 N      12  --  R4 Q17 D3
Q17 : T  -0.70c W  -0.63c S  -0.07c ( -0.3 L  -0.3) LCB   -5.24c P 10.91% WF  12.0 PSV      12 N      12  --  Q17 Q3 D3
C4  : T  -1.25c W  -1.28c S   0.03c ( -0.4 L  -0.4) LCB  -12.49c P  8.16% WF   8.0 PSV       8 N       8  --  C4 Q3 Q17
D16 : T  -1.83c W  -0.05c S  -1.78c ( -0.0 L  -0.3) LCB  -35.42c P  6.23% WF   6.0 PSV       6 N       6  --  D16 D17 E17 C17
D4  : T  -1.69c W  -1.30c S  -0.39c ( -0.5 L  -0.5) LCB  -31.68c P  5.51% WF   5.0 PSV       5 N       5  --  D4 Q3 Q17
Q16 : T  -2.12c W  -1.66c S  -0.47c ( -0.6 L  -0.6) LCB  -32.56c P  5.50% WF   5.0 PSV       5 N       5  --  Q16 D3 Q3
Q4  : T  -2.00c W  -1.23c S  -0.76c ( -0.4 L  -0.4) LCB  -68.54c P  4.90% WF   4.0 PSV       4 N       4  --  Q4 D3 Q17

: = R16
: Controller: boardsize 9
: GPU -1 finishing, processed 235 rows 235 batches
: Cleaned up old neural net and bot
: nnRandSeed0 = forTesting
: After dedups: nnModelFile0 = tests/models/g170-b6c96-s175395328-d26788732.bin.gz useFP16 false useNHWC false
: Cuda backend thread 0: Found GPU Tesla V100-SXM2-16GB memory 16945512448 compute capability major 7 minor 0
: Cuda backend thread 0: Model version 8 useFP16 = false useNHWC = false
: Cuda backend thread 0: Model name: g170-b6c96-s175395328-d26788732
: Loaded neural net with nnXLen 9 nnYLen 9
: = 
: Controller: genmove_debug b
: MoveNum: 0 HASH: 9F0B2D702FC8448C75410E097F089AEB
   A B C D E F G H J
 9 . . . . . . . . .
 8 . . . . . . . . .
 7 . . . . . . . . .
 6 . . . . . . . . .
 5 . . . . @ . . . .
 4 . . . . . . . . .
 3 . . . . . . . . .
 2 . . . . . . . . .
 1 . . . . . . . . .

koPOSITIONALscoreAREAtaxNONEsui1komi7.5
Time taken: ###
Root visits: 100
New playouts: 100
NN rows: 100
NN batches: 100
NN avg batch size: 1
PV: E4 E6 C5 C6 B6 D5
Tree:
: T -10.06c W  -8.52c S  -1.55c ( -0.8 L  -0.4) N     100  --  E4 E6 C5 C6 B6 D5
---Black(^)---
E4  : T  -6.73c W  -6.15c S  -0.58c ( -0.6 L  -0.2) LCB  -14.59c P  9.68% WF  16.5 PSV      47 N      16  --  E4 E6 C5 C6 B6 D5
E5  : T -11.28c W -10.24c S  -1.05c ( -1.2 L  -0.5) LCB  -18.18c P 26.97% WF  24.7 PSV      25 N      25  --  E5 C5 D7 D3
F5  : T  -7.67c W  -6.45c S  -1.21c ( -0.6 L  -0.3) LCB  -15.80c P 11.20% WF  18.4 PSV      16 N      18  --  F5 D5 E3 D3 D2 E4 F4
E6  : T -10.48c W  -9.39c S  -1.09c ( -0.9 L  -0.5) LCB  -22.39c P 12.51% WF  12.0 PSV      12 N      12  --  E6 E4 C5 C4 B4 D5
D5  : T -11.21c W  -8.91c S  -2.30c ( -0.8 L  -0.3) LCB  -34.59c P  7.65% WF   7.9 PSV       7 N       8  --  D5 F5 E7 F7 F8
F6  : T -14.57c W -11.09c S  -3.48c ( -1.0 L  -0.9) LCB  -61.13c P  7.40% WF   5.8 PSV       5 N       6  --  F6 D4 F4 D7
F4  : T -12.53c W  -9.15c S  -3.38c ( -1.0 L  -0.7) LCB  -85.49c P  6.98% WF   4.9 PSV       5 N       5  --  F4 D6 D4 G6
D4  : T -13.40c W -10.01c S  -3.40c ( -1.0 L  -0.8) LCB -112.18c P  5.47% WF   4.9 PSV       4 N       5  --  D4 F6 D6 F3
D6  : T -12.06c W  -8.91c S  -3.15c ( -0.7 L  -0.7) LCB -164.43c P  5.21% WF   3.9 PSV       4 N       4  --  D6 F4 F6 C4

: = E5
: Controller: showboard
: = MoveNum: 1 HASH: 38E71B648B5DE3A43B414F57C285AD76
   A B C D E F G H J
 9 . . . . . . . . .
 8 . . . . . . . . .
 7 . . . . . . . . .
 6 . . . . . . . . .
 5 . . . . X1. . . .
 4 . . . . . . . . .
 3 . . . . . . . . .
 2 . . . . . . . . .
 1 . . . . . . . . .
Next player: White
Rules: {"friendlyPassOk":false,"hasButton":false,"ko":"POSITIONAL","komi":7.5,"scoring":"AREA","suicide":true,"tax":"NONE","whiteHandicapBonus":"0"}
B stones captured: 0
W stones captured: 0
: GPU -1 finishing, processed 100 rows 100 batches
: All cleaned up, quitting
