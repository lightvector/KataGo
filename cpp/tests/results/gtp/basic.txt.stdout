= R4

= Q17

= 

= C16

= R16

= 

= E5

= MoveNum: 1 HASH: 38E71B648B5DE3A43B414F57C285AD76
   A B C D E F G H J
 9 . . . . . . . . .
 8 . . . . . . . . .
 7 . . . . . . . . .
 6 . . . . . . . . .
 5 . . . . X . . . .
 4 . . . . . . . . .
 3 . . . . . . . . .
 2 . . . . . . . . .
 1 . . . . . . . . .
Next player: White
Rules: {"hasButton":false,"ko":"POSITIONAL","komi":7.5,"scoring":"AREA","suicide":true,"tax":"NONE","whiteHandicapBonus":"0"}
B stones captured: 0
W stones captured: 0

